users/users-model.js

```diff
@@ -30,3 +31,7 @@ function findByWasherId(washerId) {
 function updateWasher(washerId, changes) {
   return db("washers").where({ washerId }).update(changes, "*");
 }
```

Comment:
Great feature to add here, wondering if there are any other endpoints that would benefits from this cars array by user id